numbers&methods.py

```diff
@@ -0,0 +1,18 @@
+monthly_budget = 15000
+
+rent = 8000
+utilities = 1500
+groceries = 2000
+transportation = 1000
+miscellaneous = 500
+
+total_expenses = rent + utilities + groceries + transportation + miscellaneous
+
+remaining_budget = monthly_budget - total_expenses
+
+print("Monthly Budget: ", monthly_budget)
+print("Total Expenses: ", total_expenses)
+print("Remaining Budget: ", remaining_budget)
+
+percentage_used = (total_expenses / monthly_budget) * 100
+print(percentage_used)
```

Comment:
print(percentage_used) this should be

print("Percentage of expenses vs monthly budger",percentage_used)

Reply:
Program has to receive arguments from command line and print the final output.